CMakeLists.txt

```diff
@@ -191,12 +191,15 @@ else()
   message(FATAL_ERROR "Required target protobuf::libprotobuf-lite or protobuf::libprotobuf is not defined")
 endif()
 
-# gdal
-set(gdal_target "")
+# unless you said you didnt want gdal we try to turn it on, if we cant we tell you
```

Comment:
i thought it would be better to default turning it ON if we can and if we couldnt we simply report that was the case. if you tell it you definitely dont want it by setting it to OFF then we dont even try

Reply:
yep will do that real quick